style.css

```diff
@@ -879,4 +1015,117 @@ body {
     margin-top: 25px;
     color: #253858;
   }
+
+  /* contact me section */
+
+  .contact-me {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: -30px;
+    background-image: url("./images/contactMe-bg.svg");
+    background-repeat: no-repeat;
+    background-size: 68%;
+    background-position: right top 40%;
+    background-color: #6070ff;
+    min-height: 100vh;
+    margin-top: -80px;
+    border-radius: 115px 0 0 0;
+    padding-top: 100px;
+  }
+
+  .contact-title {
+    color: white;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin-top: 50px;
+  }
+
+  .contact-desc {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    padding: 0 24px;
+    margin-bottom: 5px;
+  }
+
+  .form {
+    width: 100%;
+    margin: 0 24px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .form form {
+    display: flex;
+    flex-direction: column;
+    width: 85%;
+  }
+
+  .form-input {
+    height: 48px;
+    margin-bottom: 24px;
+    border-radius: 8px;
+    padding-left: 15px;
+    color: rgba(179, 186, 197, 1);
+  }
+
+  .form-input:focus {
+    border: 2px solid #396df2;
+    border-radius: 8px;
+    color: rgba(23, 43, 77, 1);
+    font-family: 'Poppins', sans-serif;
+    font-size: regular;
+    font-size: 17px;
+    line-height: 20px;
+    line-height: 78%;
+  }
+
+  #form-commnt {
+    height: 170px;
+    border-radius: 8px;
+    margin-bottom: 24px;
+    padding-left: 15px;
+    padding-top: 15px;
+    font-family: 'Poppins', sans-serif;
+    font-style: regular;
+    font-size: 15px;
+    line-height: 24px;
+    line-height: 107%;
+    color: #b3bac5;
+  }
+
+  .btn-submit {
```

Comment:
- [ ] Please 🙏 increase the `font size ` and decrease a bit 🤏  of the top-bottom padding of your submit button to make it look like the Figma design. Also please add a it more radius to the corners 👍 

_**Figma design**_

<img width="199" alt="Screenshot 2023-02-22 at 5 09 42 AM" src="https://user-images.githubusercontent.com/44798044/220487508-0a457bad-e105-41e3-8071-92d8ddcb76a2.png">

**_Your design_**

<img width="171" alt="Screenshot 2023-02-22 at 5 09 50 AM" src="https://user-images.githubusercontent.com/44798044/220487452-ee249223-d9fa-479f-9f53-8e3bbf81764b.png">